.github/workflows/samples-java-wiremock.yaml

```diff
@@ -0,0 +1,35 @@
+name: Samples Java Wiremock
+
+on:
+  push:
+    paths:
+      - samples/server/petstore/java-wiremock/**
+  pull_request:
+    paths:
+      - samples/server/petstore/java-wiremock/**
+jobs:
+  build:
+    name: Build Java Wiremock
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sample:
+          - samples/server/petstore/java-wiremock
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 11
```

Comment:
what about testing it with jdk 8 instead so as to catch the issue reported earlier?

Reply:
The version of wiremock is for Java 11.
I think this generator should support Java 11 and later only.

Reply:
if @martin-mfg  run `mvn clean verify` again no the sample, he will get an error again, right?

Reply:
the auto-generated pom.xml targets jdk 11 so it's correct to test it with jdk11 in the workflow.

if users wants the output to work with jdk8, they will need to use customized template accordingly.